clients/admin-ui/src/types/api/models/ConsentMethod.ts

```diff
@@ -6,7 +6,7 @@
  * An enumeration.
  */
 export enum ConsentMethod {
-  BUTTON = "button",
+  BUTTON = "button", // deprecated- keeping for backwards-compatibility
```

Comment:
fyi this file is autogenerated so this comment will be removed next time we autogenerate. it'd be better to put the comment in the backend, I think those comments get ported over here too

Reply:
Yup, that comment is there on the backend; I think that's why it got here in the first place 👍